### PR TITLE
changes for default & dark theme

### PR DESCRIPTION
**Pull request by apurvaubade**

### CODE REVIEW

File: src/app/menu/menu.component.scss
Comment (Ekta29yadav) [inline src/app/menu/menu.component.scss:1]: change variable name --titleColor to --headertitleColor

File: src/app/theme.config.ts
Comment (Ekta29yadav) [inline src/app/theme.config.ts:1]: --toggleColor change to --menuiconcolor

File: src/app/theme.config.ts [hunk 1]
```diff
@@ -9,8 +9,31 @@ export const THEMES = {
...
     headerLinkColor: "#F4ECFF",
-    primaryTextColor: "#000000",
+    primaryTextColor: "#494949",
     secondaryTextColor: "#8D9091",
+    
+
+
+    titleColor:"#5526f5",
+    logoPrimaryTextColor:"#1b1818",
+    logoHeadlineColor:"#79787c",
```
Comment (Ekta29yadav) [inline src/app/theme.config.ts:19]: --logoHeadlineColor change onto --headerSecondaryColor

File: src/app/theme.config.ts [hunk 2]
```diff
@@ -9,8 +9,31 @@ export const THEMES = {
...
     headerColor: "#4A227C",
     headerLinkColor: "#F4ECFF",
-    primaryTextColor: "#000000",
+    primaryTextColor: "#494949",
     secondaryTextColor: "#8D9091",
+    
+
+
+    titleColor:"#5526f5",
+    logoPrimaryTextColor:"#1b1818",
```
Comment (Ekta29yadav) [inline src/app/theme.config.ts:18]: logoPrimaryTextColor into --headerPrimaryColor

File: src/app/theme.config.ts [hunk 3]
```diff
@@ -9,8 +9,31 @@ export const THEMES = {
     navLabelColor: "#B275FF",
     headerColor: "#4A227C",
     headerLinkColor: "#F4ECFF",
-    primaryTextColor: "#000000",
+    primaryTextColor: "#494949",
     secondaryTextColor: "#8D9091",
+    
+
+
+    titleColor:"#5526f5",
```
Comment (Ekta29yadav) [inline src/app/theme.config.ts:17]: --titleColor into hedaerTitleColor

File: src/app/theme.config.ts [hunk 4]
```diff
@@ -9,8 +9,31 @@ export const THEMES = {
...
+    logoPrimaryTextColor:"#1b1818",
+    logoHeadlineColor:"#79787c",
+    logoSecondaryTextColor:"#ee8908",
+    toggleColor: "#5526f5",
+    btnPrimaryColor:"#060461",
+    headingColor:"#060461",
+    subheadingColor:"#212529",
+    searchBoxBorder:"#efefef",
+    filedBackground:"#fff",
```
Comment (Ekta29yadav) [inline src/app/theme.config.ts:26]: --filedBackground name change into InputBgColor

File: src/app/theme.config.ts [hunk 5]
```diff
@@ -22,6 +45,32 @@ export const THEMES = {
...
+    logoHeadlineColor:"#ffffff",
+    logoSecondaryTextColor:"#c3c3c3",
+    toggleColor: "#c3c3c3",
+    btnPrimaryColor:"#323232",
+    headingColor:"#ffffff",
+    subheadingColor:"#c3c3c3",
+    searchBoxBorder:"#212529",
+    filedBackground:"#323232",
+    hamBackground:"#080808",
+    menuLogoColor:"#79787c"
```
Comment (Ekta29yadav) [inline src/app/theme.config.ts:61]: --menuLogoColor into HamMenuLogoColor

**Review by Ekta29yadav — CHANGES_REQUESTED** [review]

File: src/assets/config/config.json [hunk 6]
```diff
@@ -1,12 +1,70 @@
...
+},
   "configFolder": "/assets/config/ui-config",
   "logoPath": "/assets/images/logo.svg",
-  "title": "AVSAR Learner Wallet",
-  "baseUrl": "https://ulp.uniteframework.io/ulp-bff",
-  "schemaUrl": "https://ulp.uniteframework.io/registry/api/docs/swagger.json",
-  "footerText": "AVSAR Learner Wallet",
+  "title": "Sunbird RC ELocker",
+  "baseUrl": "https://elocker.xiv.in/elocker/api/v1",
+  "schemaUrl": "https://elocker.xiv.in/elocker/api/docs/swagger.json",
```
Comment (itsvick) [inline src/assets/config/config.json:66]: @apurvaubade change these values back to the Avasar values

File: src/app/menu/menu.component.html [hunk 7]
```diff
@@ -50,11 +50,20 @@
         </div>
       </li> -->
       <li class="nav-item" (click)="logOut()">
-        <div class="float-left pr-10"><img src="assets/images/logout.png" alt=""></div>
+        <div class="float-left pr-10"><i class="fa fa-sign-out-alt float-left fs-26" aria-hidden="true"></i></div>
         <div class="pl-10">
           <div class="fs-16">{{'LOGOUT' | translate}}</div>
         </div>
       </li>
+
```
Comment (Ekta29yadav) [inline src/app/menu/menu.component.html:58]: remove blank space

File: src/app/menu/menu.component.scss [hunk 8]
```diff
@@ -13,11 +13,15 @@ select.form-control:focus{
...
 .langauge-bx{
-    border: solid 1px var(--primaryColor);
+    // border: solid 1px var(--primaryColor);
+    border: 1px solid  var(--searchBoxBorder) !important;
+
     padding: 5px 30px;
     border-radius: 7px;
     .fs-16{
-        color: var(--primaryColor);
+         color: var(--btnNameColor);
```
Comment (Ekta29yadav) [inline src/app/menu/menu.component.scss:22]: btnNameColor change variable name to btnPrimaryColor

File: src/app/menu/menu.component.ts [hunk 9]
```diff
@@ -14,18 +16,44 @@ export class MenuComponent implements OnInit {
...
+  changeTheme() {
+    if (this.ELOCKER_THEME == 'default') {
+      this.ELOCKER_THEME = "dark";
+    } else {
+      this.ELOCKER_THEME = "default";
+    }
+    this.themeService.setTheme(this.ELOCKER_THEME);
+    localStorage.setItem('ELOCKER_THEME', this.ELOCKER_THEME);
+  }
+
```
Comment (Ekta29yadav) [inline src/app/menu/menu.component.ts:50]: remove extra space

File: src/app/theme.config.ts
Comment (Ekta29yadav) [reply src/app/theme.config.ts:1]: is this done?

File: src/app/theme.config.ts [hunk 10]
```diff
@@ -1,27 +1,48 @@
...
-    tagsBackground: "#F4ECFF",
-    navLabelColor: "#B275FF",
-    headerColor: "#4A227C",
-    headerLinkColor: "#F4ECFF",
-    primaryTextColor: "#000000",
-    secondaryTextColor: "#8D9091",
+    primaryColor: '#060461',
+    secondaryColor: '#4A227C',
+    cardBackground: '#FFFFFF',
+    tagsBackground: '#F4ECFF',
```
Comment (Ekta29yadav) [inline src/app/theme.config.ts:6]: tagsBackground is this in use?

File: src/app/theme.config.ts [hunk 11]
```diff
@@ -1,27 +1,48 @@
...
+    navLabelColor: '#B275FF',
+    headerColor: '#4A227C',
+    headerLinkColor: '#F4ECFF',
+    primaryTextColor: '#494949',
+    secondaryTextColor: '#8D9091',
+    btnNameColor: '#5526f5',
+    headerPrimaryColor: '#1b1818',
+    logoHeadlineColor: '#79787c',
+    logoSecondaryTextColor: '#ee8908',
+    menuiconcolor: '#5526f5',
```
Comment (Ekta29yadav) [inline src/app/theme.config.ts:16]: menuiconcolor to menuIconColor

File: src/app/theme.config.ts [hunk 12]
```diff
@@ -1,27 +1,48 @@
...
+    btnPrimaryColor: '#060461',
+    headingColor: '#060461',
+    subheadingColor: '#212529',
+    searchBoxBorder: '#efefef',
+    hamBackground: '#f6f8fc',
+    bodyBackgroundd: '#a4a8ff3d',
```
Comment (Ekta29yadav) [inline src/app/theme.config.ts:22]: bodyBackgroundd to bgPrimaryColor

File: src/app/theme.config.ts [hunk 13]
```diff
@@ -1,27 +1,48 @@
...
+    bodysecondaryBackground: '#ffa4e042',
```
Comment (Ekta29yadav) [inline src/app/theme.config.ts:23]: bodysecondaryBackground to bgSecondaryColor

Comment (apurvaubade) [reply src/app/theme.config.ts]: resolved all

File: src/app/global-header/global-header.component.html [hunk 14]
```diff
@@ -1,33 +1,59 @@
...
-</div>
\ No newline at end of file
+  </div>
+  <div class="heading2 pb-10 mt-4" *ngIf="showTitle">AVSAR Learner Wallet</div>
+  <div class="pb-20" *ngIf="showTitle">
+    {{ "WELCOME_TO_YOUR_ACADEMIC_WALLET" | translate }}
+  </div>
+
+  <div style="position: fixed; top: 0; right: 0; padding: 10px">
+    <span (click)="changeTheme()">
```
Comment (itsvick) [inline src/app/global-header/global-header.component.html:53]: @apurvaubade Revert this file. We're not going to add option to switch the theme

File: src/app/menu/menu.component.ts [hunk 15]
```diff
@@ -1,46 +1,72 @@
...
     this.getAllLanguages();
+
+    this.ELOCKER_THEME = localStorage.getItem('ELOCKER_THEME');
+
+    if (!this.ELOCKER_THEME) {
+      localStorage.setItem('ELOCKER_THEME', 'default');
+    }
+  }
+
+  changeTheme() {
```
Comment (itsvick) [inline src/app/menu/menu.component.ts:43]: @apurvaubade remove this function if not required here

**Pull request merged**